src/main/java/org/broadinstitute/hellbender/tools/spark/sv/evidence/BreakpointEvidence.java

```diff
@@ -101,6 +102,102 @@ public String stringRep(final ReadMetadata readMetadata, final int minEvidenceMa
+            }
+            final boolean forwardStrand = words[5].equals("1");
+            final int templateSize = Integer.parseInt(words[6]);
+            final String cigarString = words[7];
+            final int mappingQuality = Integer.parseInt(words[8]);
+            final String readGroup = "Pond-Testing"; // for now, just fake this, only for testing.
+            final boolean validated = false;
+            switch(evidenceType) {
+                case "SplitRead":
+                    return new SplitRead(location, weight, templateName, fragmentOrdinal, validated,
+                            forwardStrand, cigarString, mappingQuality, templateSize, readGroup, distalTargets,
+                            readMetadata);
+                case "LargeIndel":
+                    return new LargeIndel(location, weight, templateName, fragmentOrdinal, validated,
+                            forwardStrand, cigarString, mappingQuality, templateSize, readGroup, distalTargets);
+
+                case "MateUnmapped":
+                    return new MateUnmapped(location, weight, templateName, fragmentOrdinal, validated,
+                            forwardStrand, cigarString, mappingQuality, templateSize, readGroup, distalTargets);
+
+                case "InterContigPair":
+                    return new InterContigPair(location, weight, templateName, fragmentOrdinal, validated,
+                            forwardStrand, cigarString, mappingQuality, templateSize, readGroup, distalTargets);
+
+                case "OutiesPair":
+                    return new OutiesPair(location, weight, templateName, fragmentOrdinal, validated,
+                            forwardStrand, cigarString, mappingQuality, templateSize, readGroup, distalTargets);
+
+                case "SameStrandPair":
+                    return new SameStrandPair(location, weight, templateName, fragmentOrdinal, validated,
+                            forwardStrand, cigarString, mappingQuality, templateSize, readGroup, distalTargets);
+
+                case "WeirdTemplateSize":
+                    return new WeirdTemplateSize(location, weight, templateName, fragmentOrdinal, validated,
+                            forwardStrand, cigarString, mappingQuality, templateSize, readGroup, distalTargets);
+                default:
+                    throw new IllegalArgumentException("Unknown BreakpointEvidence type: " + evidenceType);
+            }
+        }
+    }
+
+    private static StrandedInterval locationFromStringRep(final String locationStr) {
+        final String[] locationParts = locationStr.split("[\\[\\]:]");
+        final int contig = Integer.parseInt(locationParts[0]);
+        final int start = Integer.parseInt(locationParts[1]);
+        final int end = Integer.parseInt(locationParts[2]);
+        final boolean strand = locationParts.length == 4 && locationParts[3].equals("1");
```

Comment:
I was trying to get away with only one locationFromStringRep() rather than separate ones for stranded and not. This is confusing problematic for any future development so I have fixed this by having separate functions for stranded locations and unstranded.